provision-rspamd.sh

```diff
@@ -20,10 +33,56 @@ configure_dmarc()
 
 	tell_status "add Redis address, for DMARC stats"
 	echo "dmarc {
-	servers = \"$(get_jail_ip redis):6379\";
+	# Enables storing reporting information to redis
+     	reporting = true;
+  	actions = {
+        	quarantine = "add_header";
+	        reject = "reject";
+	}
 }"  | tee -a "$_etc/rspamd/rspamd.conf"
```

Comment:
The [CI test](https://travis-ci.org/msimerson/Mail-Toaster-6/builds/199783457) failure shows there's an unterminated quote. Doh! A quoted string has to have all the quotes within escaped. Which is rather unsightly, making the code hard to read. Better to just refactor as a heredoc:

```sh
	tee -a "$_etc/rspamd/rspamd.conf" <<EO_DMARC
dmarc {
	# Enables storing reporting information to redis
	reporting = true;
	actions = {
		quarantine = "add_header";
	        reject = "reject";
	}
}
EO_DMARC
```

It's much easier to read what's inside the block when it's not polluted with \\"all\\" them \\"escaped quotes\\".